packages/state-transition/src/epoch/computeUnrealizedCheckpoints.ts

```diff
@@ -33,8 +34,8 @@ export function computeUnrealizedCheckpoints(
 
     // Clone state and use progressive balances
     else {
-      // Clone state to mutate below           false = do not transfer cache
-      stateRealizedCheckpoints = state.clone(false);
+      // Clone state to mutate below         true = do not transfer cache
+      stateRealizedCheckpoints = state.clone(true);
```

Comment:
This is the main fix: to prevent transferring the cache to this temporary state `true` must be passed, not `false`

Reply:
👍